app/controllers/concerns/idv/threat_metrix_concern.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Idv
+  module ThreatMetrixConcern
+    THREAT_METRIX_DOMAIN = 'h.online-metrix.net'
+    THREAT_METRIX_WILDCARD_DOMAIN = '*.online-metrix.net'
+
+    def override_csp_for_threat_metrix
+      return unless Rails.env.production?
+
```

Comment:
maybe remove this so we can run tests in our specs? And rely on the feature flag below being correctly configured?
```suggestion
```